assets/src/edit-story/components/canvas/multiSelectionMovable.js

```diff
@@ -273,7 +273,7 @@ function MultiSelectionMovable({ selectedElements }) {
       verticalGuidelines={canSnap ? [0, canvasWidth / 2, canvasWidth] : []}
       elementGuidelines={otherNodes}
       snapGap={canSnap}
-      isDisplaySnapDigit={true}
+      isDisplaySnapDigit={false}
```

Comment:
What was this doing, that is now not necessary?

Reply:
It prints actual distances, which was pretty nice. Unfortunately, at lower resolutions (when our canvas is resized responsively) these numbers are scaled and not in our "data pixels". So it's better to disable for now. Hopefully eventually we can specify our "formatter" callback where we could reconcile this.

Reply:
Moveable had a bug that still showed numbers when passing `false`

#884 temporarily turned numbers on again.

Now the bug seems to be fixed.

See https://github.com/daybrush/moveable/issues/200 for background